README.md

```diff
@@ -4,31 +4,33 @@ A Neural Chess AI in Python
 
 ## ♙ Description
 
-This aims to be a simple and inefficient chess engine programed in Python for learning about AI, PyTorch and dataset generation
+This aims to be a simple and efficient chess engine programed in Python for learning about AI, PyTorch and dataset generation.

```

Comment:
I meant *inefficient* because i am using python thus is inherently slow, doing this in another language could be 80x faster, maybe i could word this differently

Reply:
Thanks for the pull request and tanking interest in the project!